test/functional/wallet_simulaterawtx.py

```diff
@@ -125,5 +167,183 @@ def run_test(self):
         assert_raises_rpc_error(-8, "One or more transaction inputs have been spent already", w1.simulaterawtransaction, [tx1, tx2])
         assert_raises_rpc_error(-8, "One or more transaction inputs have been spent already", w2.simulaterawtransaction, [tx1, tx2])
 
+
+
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

nit: Extra blank lines can be removed.